storage/innobase/lock/lock0lock.cc

```diff
@@ -6547,6 +6551,18 @@ void lock_trx_release_locks(trx_t *trx) /*!< in/out: transaction */
 
   trx_mutex_enter(trx);
 
+#ifdef WITH_WSREP_OUT
```

Comment:
similar to above, can this block be removed?

Reply:
yes, removed.